kahuna/public/js/image/view.html

```diff
@@ -188,66 +188,75 @@
-                          e:form="creditEditForm">
-
-                        <span ng:if="ctrl.metadata.credit">
-                            <a ui:sref="search.results({query: (ctrl.metadata.credit | queryFilter:'credit')})">{{ctrl.metadata.credit}}</a>
-                        </span>
-                        <span ng:if="! ctrl.metadata.credit">
-                            Unknown (click ✎ to add the credit)
+                <dl class="flex-container full-width"
+                    ng:if="ctrl.metadata.credit || ctrl.userCanEdit">
+                    <dt class="image-info__credit image-info__wrap image-info__group--dl__key metadata-line__key">Credit</dt>
+                    <dd class="image-info__wrap image-info__group--dl__value metadata-line__info">
+                        <button class="image-info__edit"
+                            ng:if="ctrl.userCanEdit"
+                            ng:click="creditEditForm.$show()"
+                            ng:hide="creditEditForm.$visible"
+                        >✎</button>
+
+                        <span class="metadata-line__info"
+                              editable-text="ctrl.metadata.credit"
+                              e:typeahead="credit for credit in ctrl.credits($viewValue) | limitTo:8"
+                              onbeforesave="ctrl.updateMetadataField('credit', $data)"
+                              e:ng-class="{'image-info__editor--error': $error, 'image-info__editor--saving': creditEditForm.$waiting}"
+                              e:form="creditEditForm">
+
+                            <span ng:if="ctrl.metadata.credit">
+                                <a ui:sref="search.results({query: (ctrl.metadata.credit | queryFilter:'credit')})">{{ctrl.metadata.credit}}</a>
+                            </span>
+                            <span ng:if="! ctrl.metadata.credit">
+                                Unknown (click ✎ to add)
+                            </span>
                         </span>
-                    </span>
-                </dd>
+                    </dd>
+                </dl>
 
 
-                <dt ng:if="ctrl.metadata.subLocation || ctrl.metadata.city || ctrl.metadata.state || ctrl.metadata.country" class="image-info__group--dl__key metadata-line__key">Location</dt>
-                <dd ng:if="ctrl.metadata.subLocation || ctrl.metadata.city || ctrl.metadata.state || ctrl.metadata.country" class="image-info__group--dl__value metadata-line__info">
-                    <span ng:repeat="prop in ['subLocation', 'city', 'state', 'country']" ng:if="ctrl.metadata[prop]">
-                        <span class="metadata-line__info">
-                            <a ui:sref="search.results({query: (ctrl.metadata[prop] | queryFilter:ctrl.locationFieldMap[prop])})">{{ctrl.metadata[prop]}}</a></span><span ng:if="! $last">,</span>
-                    </span>
-                </dd>
+                <dl class="flex-container full-width"
```

Comment:
> Isn't it ont dl with multiple items?

[Yes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl). Only `dt` and `dd` are valid according to the spec. Reverted this.